src/Symfony/Component/HttpFoundation/BinaryFileResponse.php

```diff
@@ -201,9 +201,6 @@ public function prepare(Request $request)
 
         $this->ensureIEOverSSLCompatibility($request);
 
-        $this->offset = 0;
```

Comment:
i think we should keep this in case of multiple calls to `prepare()`.

Technically initializing the properties with default values can break some edgy  code, relying on the the lazy null. Perhaps better to default on the calling side: `$this->offset ?? 0`?